RentallCarsAPI/Controllers/CarController.cs

```diff
@@ -18,86 +16,175 @@ public class CarController : Controller
         [HttpPost]
         public IActionResult Create(CarRequest model)
         {
-            var oResponse = new Response();
+            var response = new Response();
+           
+            var cars = GetAll();
+            if (cars == null)
+            {
+                response.Message = "File reading failed";
+                return BadRequest(response);
+            }
+            if (!ValidateParams(model))
+            {
+                response.Message = "Invalid Transmition or Mark";
+                return BadRequest(response);
+            }
+            var car = new Car();
+            car.Id = ++cars.Last().Id;
+            car.Transmition = model.Transmition;
+            car.Mark = model.Mark;
+            car.Model = model.Model;
+            car.Doors = model.Doors;
+            car.Color = model.Color;
```

Comment:
Qué te parece si instanciamos la clase `Car` de forma declarativa ? [Acá](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/how-to-initialize-objects-by-using-an-object-initializer) tenemos documentación para mirar 
```suggestion
            var car = new Car
            {
                Id = ++cars.Last().Id,
                Transmition = model.Transmition,
                Mark = model.Mark,
                Model = model.Model,
                Doors = model.Doors,
                Color = model.Color
            };
```

Reply:
buenas noches Darian!! me parece excelente. nunca lo hice, pero lo había visto, y observando la documentación q me enviaste a ver me quedo mucho mas que claro. tengo q ayornarme un poco con la programación declarativa! una consulta... es mi primer PR. y acá veo  q en la observación q me hiciste me da la posibilidad de commitear. puede ser¿?

Reply:
Hola Matias! Sí, está la posibilidad de utilizar esa opción!

Reply:
excelente.. ya lo modifique yo igualmente, en breve subo el commit